### ISSUE TITLE
The lights from kOS illuminate planet surfaces very far away that they shouldn't.

### ISSUE THREAD

**Issue by Dunbaratu** [KSP-KOS/KOS#2171]
The lights like the cherry light will illuminate the planet surface from very far away up in orbit in a way that looks kind of silly.  I never really knew enough about that part of the mod to know how to fix that, but I came across this post when looking up something else, and it contained this nugget of information that I think may be relevant here:

> Layer 10 Scaled Scenery       Notes on lights. If the  lights of your craft  illuminate a planet surface  while you are orbiting. They need to be set to ignore this layer (see this post) thanks to @Nils277 for the discovery (it's bugged a particular project of mine forever) 

(this was from the KSP forum post here: https://forum.kerbalspaceprogram.com/index.php?/topic/84273-unity-layers-and-tags-updated-09-march-2017/

**Comment by pgodd** [comment]
Hmm. I will have a look.

**Comment by pgodd** [comment]
I have the project files still set up.

**Comment by pgodd** [comment]
Is there a list of layer names? Naturally I'm not developing inside the kerbal engine so I don't have a "planets" layer per se.

**Comment by Dunbaratu** [comment]
Yeah that's a really annoying problem.  I don't think there is a list. Or if there is it's well hidden where modders can't find it.  I can only go from posts like this one in the forum.  One serious frustration I have is that the layer mask is a 32-bit mask, and KSP is already using layers 0 through 31, leaving none open for modders to do anything with.

**Comment by pgodd** [comment]
Well that’s more information again.
I’m still fighting with how to convert emissive masks to dds but with any luck I can put both to bed tomorrow

**Comment by pgodd** [comment]
Found layers information.
https://wiki.kerbalspaceprogram.com/wiki/API:Layers
It looks simple enough.
